### ISSUE TITLE
tuplet numbers and accents

### ISSUE THREAD

**Issue by hkiel** [leesavide/abcm2ps#60]
Following example shows the behaviour for tuplets with accents. When the tuplet number is on the same side as the accent, the tuplet is drawn nearer than the accent which looks odd. 
```
X:1
K:C
[I:tuplets 0 0 0 2](3!>!g!>!a!>!b (3!>!G!>!A!>!B | [I:tuplets 0 0 0 1] (3!>!g!>!a!>!b (3!>!.g!>!.a!>!.b
```

**Comment by moinejf** [comment]
You wrote "the tuplet is drawn nearer than the tuplet". Maybe it should be "the tuplet is drawn nearer than the accent".
Well, I have no solution for this problem: all note related decorations are drawn after the tuplets. To fix this problem, I should create an other class of decorations, and this is a big change I will not do in abcm2ps.

**Comment by hkiel** [comment]
Staccato dots, however, are drawn directly next to the notes, so I thought there might be a way to fix the accents as well.

**Comment by moinejf** [comment]
This issue should be fixed by the commit [e2b8d2f], but I don't know if this raises some other problems...
